dev/com.ibm.ws.security.token.ltpa/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -26,19 +26,26 @@ keysFileName.desc=The path to the file that contains the LTPA primary keys, whic
 
 keysPassword=LTPA primary keys password
 keysPassword.desc=Password for the LTPA primary keys. The best practice is to encrypt the password by using the securityUtility tool.
+
 expiration=LTPA token expiration
 expiration.desc=Amount of time after which a token expires in minutes.
```

Comment:
expires in minutes - should be in seconds... customer can specify in second, minute